telegram/ext/filters.py

```diff
@@ -901,6 +911,8 @@ def filter(self, message: Update) -> bool:
             :attr:`telegram.Message.group_chat_created`,
             :attr:`telegram.Message.supergroup_chat_created` or
             :attr:`telegram.Message.channel_chat_created`.
+        connected_website: Messages that contain
+            :attr:`telegram.Message.connected_website`.
```

Comment:
thats not introduced here, mistake or fix?

Reply:
The filter already existed, just added it to the docstring